packages/actions/index.ts

```diff
@@ -1143,3 +1160,64 @@ export const getTransferLimitLevel = async (userAddress, store) => {
 export const exceedsTransferLimit = (amount: number, transferLimit: string, ethExchange: string, ethSentPastWeek: number) => {
   return (ethSentPastWeek * Number(ethExchange) + amount) > Number(transferLimit)
 }
+
+export const sendEmailTx = (inviteTx: InviteTransaction) => {
+  return async (dispatch, getState) => {
+    const { privateKeyBuffer } = getUser(getState())()
+
+    return creditProtocol.sendEmailTx(inviteTx, privateKeyBuffer)
+    .catch(err => {
+      dispatch(displayError(debtManagement.pending.error))
+      throw new Error('Unable to send email transaction: ' + err)
+    })
```

Comment:
:memo: `sendEmailTx()` is asymmetrically structured to its siblings above in that it uses promise chains instead of async/await and uses a different variable name for the caught error. Would help with readability for each of these actions, which have very similar logic, to follow a consistent code style / pattern.